model/commitqueue/github_pr_sender.go

```diff
@@ -76,6 +76,8 @@ func (s *githubPRLogger) Send(m message.Composer) {
 
 	s.sendPatchResult(msg)
 	if !msg.PatchSucceeded {
+		s.ErrorHandler(errors.New("not proceeding with merge for failed patch"), m)
+		s.ErrorHandler(dequeueFromCommitQueue(msg.ProjectID, msg.PRNum), m)
```

Comment:
if the patch doesn't succeed dequeue the item so the queue doesn't block waiting on new commits to the PR.